### DISCUSSION TITLE
Get-PnPTeamsChannelMessage : Forbidden

### DISCUSSION THREAD

**Discussion by Aharon-Bensadoun** [pnp/powershell#1614] [General]
Hi, 
I'm trying to run the command Get-PnPTeamsChannelMessage -Team Test-Channel Test and i receive:
Get-PnPTeamsChannelMessage : Forbidden.
I miss something ?
Both the team and the channel exist and i have permission on it.

**Comment by LeonArmston** [comment]
Hi @Aharon-Bensadoun 

It seems to work fine for me with this approach

![image](https://user-images.githubusercontent.com/12968962/153415600-1fdfbabb-4ac6-4c04-bfff-7ddde5b537c9.png)

I am an owner of Team and have Microsoft Graph API: Group.ReadWrite.All permissions to my delegated user account which I used to authenticate.

**Reply by Aharon-Bensadoun** [reply]
Hi Leon, 
I try to connect as you sent but I receive an error:
![image](https://user-images.githubusercontent.com/56069135/153744103-2987921c-8a74-4e61-94f0-e6b125bdc584.png)
![image](https://user-images.githubusercontent.com/56069135/153744112-a802094a-6b46-48d9-a564-137602392b61.png)

my user is of course owner on the team and has Group.ReadWrite.All permissions

**Comment by Aharon-Bensadoun** [comment]
Update !

With this way it's working 👍 
![image](https://user-images.githubusercontent.com/56069135/153744593-f44d1e20-d3dd-4533-8841-871323caef26.png)
